includes/utilities.inc

```diff
@@ -505,13 +505,13 @@ function islandora_solr_truncate_field_display($display_values, $max_length, $ad
 
     $updated_display_values
       = "<span class='toggle-wrapper'>" .
-      t("<span>!value !separator !show_more</span>", array(
+      t("<span>!value !show_more</span>", array(
         '!separator' => $separator,
```

Comment:
@nhart That makes more sense but it is currently I could only find it called using the `<br />` separator (on line [345](https://github.com/BrandonMorr/islandora_solr_search/blob/1361e18bad55ca19429311cb1a2bb40aabedb540/includes/utilities.inc#L345)).

Currently you get the **Show more** on a separate line. This will change the default display to have that **Show more** on the same line as the truncated item.

Perhaps that was the desired outcome, I just wasn't sure.